VK/VK/Model/Group/GroupInfo.swift

```diff
@@ -3,7 +3,7 @@
 
 import Foundation
 
-// GroupInfo
+/// GroupInfo
```

Comment:
```suggestion
/// 
```
либо все на русском, либо все на английском, миксовать не надо